src/main/java/org/broadinstitute/consent/http/resources/MetricsResource.java

```diff
@@ -1,16 +1,19 @@
 package org.broadinstitute.consent.http.resources;
 
 import com.google.inject.Inject;
+
+import org.broadinstitute.consent.http.models.DatasetMetrics;
 import org.broadinstitute.consent.http.models.Type;
 import org.broadinstitute.consent.http.service.MetricsService;
 
 import javax.ws.rs.GET;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 
-@Path("/metrics")
+@Path("api/metrics")
```

Comment:
Okay, so I added this because on the front end, when we call the api in the ajax method we get the api url from the config file, and the api url in the config file has /api at the end, so this does not work with that url. It returns 404 path not found.  How should I get around this issue? in consent, I could remove the class path @Path("/metrics") and update each of the pre-existing apis' path to include /metrics at the front and then update the new datasetmetrics api to include api/metrics at the front so that just that one is accessible and the others don't break, or I could hardcode the api url with no '/api' at the end in the ajax method.

Reply:
The problem you're describing is a UI problem that we need to solve there, not here. This code should follow the same pattern we use for all metrics calls.

Reply:
Okay, the other metric apis do not have ajax methods and are not called directly from the UI. So, I'm not sure what pattern to follow, I could add another api url (without /api at the end) to the config file and retrieve that one for this specific ajax method

Reply:
That's correct. The UI makes some assumptions that aren't true any longer, so we need to update our UI code to handle this new case. The built in assumption being that all ajax calls are prefixed with `api` and that's just not true any longer. For this case, we'll need to do some config refactoring in the UI.

Reply:
Okay, thanks!